code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -32,6 +32,8 @@
 
 	var/list/target_rules = list()
 
+	var/obj/machinery/atmospherics/unary/vent_pump/entry_vent // The vent to target for ventcrawling
```

Comment:
get rid of this, this should not be a var on the mob, like I said put it in the proc

Reply:
It's used outside of the proc too

Reply:
You can implement it in a way that it doesn't have to be if you make all of it part of one proc. I don't know how much rewriting is reasonable but having this extra var just for when mobs are moving toward a vent seems odd (isn't that what `target` is normally for anyway?)

Reply:
Okay, entry_vent is removed and replaced with a boolean that lets the mob ventcrawl, no more null setting for vents